packages/components/src/public/QueryModel/SaveViewModal.tsx

```diff
@@ -223,18 +232,20 @@ export const SaveViewModal: FC<Props> = memo(props => {
                                 <span className="margin-left">Make this grid view available to all users</span>
                             </div>
                         )}
-                        {isProductProjectsEnabled(moduleContext) && isAppHomeFolder(container, moduleContext) && canEditShared && (
-                            <div className="form-check">
-                                <input
-                                    className="form-check-input"
-                                    type="checkbox"
-                                    name="setInherit"
-                                    onChange={toggleInherit}
-                                    checked={canInherit}
-                                />
-                                <span className="margin-left">Make this grid view available in child folders</span>
-                            </div>
-                        )}
+                        {isProductProjectsEnabled(moduleContext) &&
+                            isAppHomeFolder(container, moduleContext) &&
+                            canEditShared && (
+                                <div className="form-check">
+                                    <input
+                                        className="form-check-input"
+                                        type="checkbox"
+                                        name="setInherit"
+                                        onChange={toggleInherit}
+                                        checked={canInherit}
+                                    />
+                                    <span className="margin-left">Make this grid view available in all Projects</span>
```

Comment:
Heh, I feel like I heard we're actually going the other way and going to be referring to "projects" as "folders". Maybe that's too far down the road. Might be worth asking @labkey-keegang about.

Reply:
We are indeed planning to make this switch, so this will also get updated at that time (currently slated for 24.3).  In the meantime (and sorry for the nit-picking), are we intending for projects to always be capitalized?  Connecting with out German roots?